reana/cli.py

```diff
@@ -1923,6 +1937,13 @@ def python_unit_tests(component):  # noqa: D301
             cmd_activate_venv = "source  ~/.virtualenvs/_{}/bin/activate".format(
                 component
             )
+            if does_component_need_db(component):
+                run_command(
+                    "docker stop reana-postgres\n"
```

Comment:
In case people run several commands:

```console
$ reana-dev python-unit-tests -c r-j-controller
$ reana-dev python-unit-tests -c r-server
```

and in case some of these processes hang, it'll good to use different DB containers.

What about giving them the same name as for Python venv? (like `__reana-job-controller`)

Reply:
> docker: Error response from daemon: Invalid container name (_reana-server), only [a-zA-Z0-9][a-zA-Z0-9_.-] are allowed.

Reply:
`reana-job-controller__` can be used though.

Reply:
... or just start the container name with "postgres" which would also better indicate the nature of the running container... giving `postgres__reana-job-controller`.

BTW we use `postgres:9.6.2` with pinned version when installing REANA, so let's use the same pinned version for the tests too.